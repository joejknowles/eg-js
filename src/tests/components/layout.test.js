import Component from '../../components/layout';

import React from 'react';
import ReactTestUtils from 'react-addons-test-utils';

const shallowRenderer = ReactTestUtils.createRenderer();

it('renders layout as before', () => {
  const component = shallowRenderer.render(<Component children={ <div className="child" /> } />);
  expect(component).toMatchSnapshot();
});
