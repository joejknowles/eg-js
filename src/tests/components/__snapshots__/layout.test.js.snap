exports[`test renders layout as before 1`] = `
<div
  className="App">
  <div
    className="App-header">
    <h2>
      <a
        onClick={[Function]}
        style={
          Object {
            "cursor": "pointer",
          }
        }>
        EG Tech Test
      </a>
    </h2>
  </div>
  <div
    className="content">
    <div
      className="child" />
  </div>
</div>
`;
